power.py

```diff
@@ -597,8 +597,8 @@ def show_help():
     print('   -h            Show this help')
     print('   -i            Show information on available freqs, C-States, etc')
     print('   -l            List information on each core')
-    print('   -M <freq>     Set core maximum frequency')
-    print('   -m <freq>     Set core minimum frequency')
+    print('   -M <freq>/max     Set core maximum frequency')
+    print('   -m <freq>/max     Set core minimum frequency')
```

Comment:
This changes the alignment of the indentation of the description. I would recommend:
   -M \<freq>     Set core maximum frequency (can also use keyword 'max')
and
   -m \<freq>     Set core minimum frequency (can also use keyword 'max')
This keeps the indentation consistent, and will also allow for "min", and "p1" in the future.
The rest looks good, thanks!